pydov/types/interpretaties.py

```diff
@@ -222,28 +200,6 @@ def __init__(self):
         super(HydrogeologischeStratigrafieLaag, self).__init__(
```

Comment:
Is this correct, the HydrogeologischeStratigrafieLaag in the class of InformeleStratigrafie?